src/helpers.rs

```diff
@@ -423,11 +407,16 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 TimedOut => "ETIMEDOUT",
                 AlreadyExists => "EEXIST",
                 WouldBlock => "EWOULDBLOCK",
-                _ => throw_unsup_format!("The {} error cannot be transformed into a raw os error", e)
+                _ => {
+                    throw_unsup_format!("The {} error cannot be transformed into a raw os error", e)
+                }
```

Comment:
Maybe we could set `match_arm_blocks = false` to avoid things like this?

Reply:
Interestingly this still gets a block, even with `match_arm_blocks = false`? Any idea why?

Reply:
maybe rustfmt bailed out further up in the ast here out of other reasons?